napari/__init__.py

```diff
@@ -36,6 +36,7 @@
     view_points,
     view_vectors,
 )
+from .movie.naparimovie import Movie
```

Comment:
I'd put an `__init__.py` inside the folder `movie` and rename `naparimovie.py` to just `movie.py` (no need for the extra napari now this will live in the main repo. Inside the `__init__.py` you can do from `from .movie import Movie`, and then inside this file you can also do `from .movie import Movie` both of which are a little cleaner.

I'm also fine with switching `Movie` to `Animate`. Maybe then I'd use `animation` for the folder and `animate.py` and `Animate` for the file and class, so in this file you'd do `from .animation import Animate` and inside the `__init__.py` in the folder `from .animate import Animate`. What does @jni think?